pkg/cluster/k8sres.go

```diff
@@ -290,6 +290,26 @@ func nodeAffinity(nodeReadinessLabel map[string]string) *v1.Affinity {
 	}
 }
 
+func generatePodAffinity(team string, version string) *v1.Affinity {
+	// generate pod anti affinity to avoid multiple on instances on the same node
+	matchLabels := make(map[string]string)
+
+	matchLabels["application"] = "spilo"
```

Comment:
this should come from `cluster_labels`  parameter in operator configuration

Reply:
Ok, I'll try to change it to use the `cluster_labels` instead the hardcoded once.

Reply:
I've changed the code to use the `cluster_labels`.